### DISCUSSION TITLE
视频清晰度不能固定

### DISCUSSION THREAD

**Discussion by tangrsgithub** [ywmoyue/biliuwp-lite#167] [Q&A]
我手动选择1080p，然后播放了几个视频之后就会自动变回4k，然后我又要手动选1080p，要不出个固定清晰度选择的设置吧。实测网页端手动选定1080p之后不会自动变回4k，能够固定，不会这样反复横跳。

**Comment by ywmoyue** [comment]
有这种事？我看代码里面手动设置分辨率是会保存对应分辨率的id的，能否给下描述情况中对应的视频bv号

**Reply by tangrsgithub** [reply]
BV1KM4y1J7QW这个是1080p，BV1dj411U744这个打开就变4k，不过是随机的，有时候下一个就变，有时候隔十几个才变，视频参考：[https://imgur.com/a/k2ymOaj](url)
